src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2033,6 +2045,16 @@ components:
               description: This cursor allows forward pagination, by providing the cursor from the previous request.
               type: string
               nullable: true
+            uptime_range:
```

Comment:
/How about `D90`, `D30`, ..., `H12`? Identifiers (enum values) cannot start with digit on any sane platform so this would result in cleaner code.